python/tchannel/event.py

```diff
@@ -40,8 +40,11 @@
     after_receive_request=0x21,
     before_receive_response=0x30,
     after_receive_response=0x31,
-    after_receive_error=0x40,
-    after_send_error=0x41,
+    after_receive_system_error=0x40,
+    after_receive_system_error_per_attempt=0x41,
```

Comment:
Defined in the doc https://github.com/uber/tchannel/blob/master/docs/metrics.md
anything with per_attempt means considering each attempt situation in the retry. Anything without per_attempt means only consider the number of times to report to user.